### PR TITLE
[Issue 3409] Support '--help' option of Pulsar command-line tools

### PR DESCRIPTION
**Pull request by murong00**
Motivation

Fixes https://github.com/apache/pulsar/issues/3409

Modifications

Support '--help' option of Pulsar command-line tools and modify usage of 'pulsar-perf'.

### CODE REVIEW

File: bin/pulsar-perf [hunk 1]
```diff
@@ -93,7 +93,7 @@ or command is the full name of a class with a defined main() method.
 
 Environment variables:
    PULSAR_LOG_CONF               Log4j configuration file (default $DEFAULT_LOG_CONF)
-   PULSAR_CLIENT_CONF          Configuration file for client (default: $DEFAULT_CLIENT_CONF)
+   PULSAR_CLIENT_CONF            Configuration file for client (default: $DEFAULT_CLIENT_CONF)
```
Comment (ambition119) [inline bin/pulsar-perf]: The indentation here is also fix by the way.
[help](https://github.com/apache/pulsar/blob/4aef300a2d50cc1e79622fc3120bb1e7a8f8335e/bin/pulsar-perf#L90)

Comment (murong00) [reply bin/pulsar-perf]: Changed, thanks!

Comment (ambition119) [reply bin/pulsar-perf]: > Changed, thanks!

well done.

**Comment by jiazhai** [comment]
retest this please

**Comment by sijie** [comment]
run cpp tests

**Comment by sijie** [comment]
run cpp tests

**Comment by merlimat** [comment]
run cpp tests

**Comment by sijie** [comment]
run cpp tests
run java8 tests

**Comment by sijie** [comment]
run cpp tests

**Comment by merlimat** [comment]
run cpp tests

**Comment by murong00** [comment]
@merlimat @sijie The cpp tests failed due to a minor bug in `pulsar-daemon`, PTAL, thanks!

**Comment by murong00** [comment]
run java8 tests

**Comment by sijie** [comment]
run java8 tests

**Comment by merlimat** [comment]
Merged in 2.3.1 at 043f32f

**Pull request merged**